controllers/admin/AdminSelfUpgradeController.php

```diff
@@ -311,13 +249,6 @@ public function init()
             );
         }
```

Comment:
This block can be removed, a comment has been added on https://github.com/PrestaShop/autoupgrade/pull/1117